CoreFoundation/String.subproj/CFString.h

```diff
@@ -165,7 +165,7 @@ struct __CFConstStr {
         uint8_t _pad[4];
     } _base;
     uint8_t *_ptr;
-    uint32_t _length;
+    unsigned long _length;
```

Comment:
This must remain the same since the emission is determined by what clang itself emits (and has emitted for previously compiled libraries) this layout is paired not only to the emission of `@""` but also `CFSTR("")`